### ISSUE TITLE
question about pedestrians 

### ISSUE THREAD

**Issue by hrz2000** [autonomousvision/plant#12]
Thank you for your excellent work, but on what basis did you not add pedestrians to the plant? It doesn’t seem to be difficult from an implementation perspective.

**Comment by kashyap7x** [comment]
You are correct, it is unlikely it would be difficult from an implementation perspective. There are very few pedestrians in CARLA's Leaderboard 1.0 and related benchmarks like Longest6. This is also true for other aspects of the full driving task such as following stop signs and circumventing static objects, which can be ignored on CARLA while still achieving good scores.

For PlanT, we chose to focus on vehicle collisions, since it has the largest impact on driving scores in current benchmarks.

**Comment by hrz2000** [comment]
Thanks for your reply!
